strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFile/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿using ReadStringFromResourceFile;
+
+ResourcesManager rmEnglish = new ResourcesManager(@"ReadStringFromResourceFile.Resources.Texts.English");
+Console.WriteLine(rmEnglish.GetString("GREETINGS_TEXT"));
+
+var logoff = rmEnglish.GetString("LOGOFF_2");
```

Comment:
We should probably at least display this string, otherwise we can change this to just be a discard